client/app/components/Input/InputRadio.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../styles/_global.scss";
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+
+  .content {
+    margin-right: 15px;
```

Comment:
You can replace this with `$size-16` which comes from [here](https://github.com/ifmeorg/ifme/blob/master/app/assets/stylesheets/base/_sizes.scss#L10).